tests/unit/test-firefox/test.firefox.js

```diff
@@ -47,7 +48,19 @@ describe('firefox', () => {
       }));
     }
 
-    function runFirefox({profile = fakeProfile, ...args}: Object = {}) {
+    // TODO: add pipes to Flow type after https://github.com/facebook/flow/issues/2405 is fixed
+
+    type RunFirefoxOptions = {
+      profile?: typeof FirefoxProfile,
+      args?: Array<any>,
```

Comment:
I think `args` can be deleted because that's actually a property name that gets passed to the function. The `...args` syntax will spread the object into specific properties (none of which are named `args`). 

Also, I think it's ok in this case *not* to define each specific property because it's just passing through properties to the `firefox.run()` call. In other words, if one of the tests calls `runFirefox()` with invalid properties, the `FirefoxRunOptions` type will catch them. 

Thus, you can edit the TODO comment and just say something like "This object should accept dynamic properties since those are passed to firefox.run()"